contentcuration/contentcuration/frontend/shared/data/constants.js

```diff
@@ -7,7 +7,7 @@ export const CHANGE_TYPES = {
 };
 
 // Tables
-export const CHANGES_TABLE = '__changesForSyncing';
+export const CHANGES_TABLE = 'changesForSyncing';
```

Comment:
Why have you changed the name of the changes table here? Please undo this.

Reply:
Is there any specific reason use two underscores except of a convention?

Dexie Observable seems to ignore tables with two underscores. If I keep them, [`changes`](https://github.com/learningequality/studio/pull/2699/files#diff-ecc1bd1b613f8bed6daedaa78272bbef26e1500a082296ce884702f5d5401e5fR8) won't contain any change objects related to our changes table hence making it impossible to listen for its updates.

Reply:
Interesting - hadn't realized that the preceding underscores was an issue, couldn't find anything about it in the Dexie Observable documentation!

We definitely should keep this change then - could you also change the table name below too, as it will suffer from a similar problem, and may explain some of the issues we have seen with the change locks.

Reply:
Good find @MisRob !

Reply:
And just in time - so we don't have to use the nuclear option on people's IndexedDBs!

Reply:
I've just checked it one more time and even one underscore causes the same problems. I assume that it might have something to do with Dexie's internal tables that start with `_` (like `_changes`).

Reply:
> could you also change the table name below too

Will do

Reply:
@MisRob I found this [bit of code](https://github.com/dfahlander/Dexie.js/blob/v2.0.4/addons/Dexie.Observable/src/override-parse-stores-spec.js#L22), so you're correct-- tables starting with an underscore are excluded from observation!

Reply:
Thanks for checking it, @bjester